lib/msf/util/dot_net_deserialization.rb

```diff
@@ -12,6 +12,8 @@ module DotNetDeserialization
   DEFAULT_GADGET_CHAIN = :TextFormattingRunProperties
 
   def self.encode_7bit_int(int)
+    return "\x00".b if int == 0
```

Comment:
This fixed a bug when encoding a length-prefixed string that was empty. An empty string has a length of 0 and should just be `\x00`.